src/runtime/components/forms/Input.vue

```diff
@@ -154,10 +154,7 @@ export default defineComponent({
   setup (props, { emit, slots }) {
     const { ui, attrs } = useUI('input', props.ui, config, { mergeWrapper: true })
 
-    const { emitFormBlur, emitFormInput, formGroup } = useFormGroup(props)
-    const color = computed(() => formGroup?.error?.value ? 'red' : props.color)
-    const size = computed(() => formGroup?.size?.value ?? props.size)
-    const id = formGroup?.labelFor
+    const { emitFormBlur, emitFormInput, size, color, inputId, name } = useFormGroup(props)
```

Comment:
```suggestion
    const { emitFormBlur, emitFormInput, size, color, inputId } = useFormGroup(props)
```

Reply:
Why is the `name` not used to bind on the `<input>`?

Reply:
I went too fast, fixed it here https://github.com/nuxt/ui/pull/704/commits/fb2442b1eb0101fb2ac9461491f66d02789bb3d4